Include barcodes from AFH and Workplace Outbreaks

kschwabefry opened the pull request:
Include barcodes from Adult Family Homes and Workplace Outbreaks on the
switchboard. This project is a "classic" RedCAP project, such that it is not
longitudinal and does not have events or arms. The RedCAP API call to fetch
events will fail for classic projects, and we don't actually need to provide
and arm or event to construct the deep link URL into the PCDEQC instrument for
records in this classic project. Include barcode variables
(core_collection_barcode and return_collection_barcode) from the new project.

Constructed the RedCAP project using the new style, without explicitly passing
the token. This required updating the ID3C library, and I refreshed the
versions of other dependencies along with it.

kschwabefry commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Add AFH study to switchboard](https://trello.com/c/Frt7uwBM/964-add-afh-study-to-switchboard)

jccraft-uw reviewed (approved):
This makes a lot of sense. See what needs to change when you rebase in the env.d changes that got merged this morning.

kschwabefry commented:
Rebased on master (and dropped my pipefile.lock dependency changes in favor of https://github.com/seattleflu/scan-switchboard/commit/52f21c519dc1c411ea449ad6bddb93a998cbb45f)

kschwabefry commented:
Deployed.
